From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Tue, 19 Jun 2018 11:17:34 +0200
Subject: [PATCH] {MAYBE UPS} ARM syscall: wire up missing io_pgetevents

scripts/checksyscalls.sh reports:

    error: #warning syscall io_pgetevents not implemented [-Werror=cpp]
---
 arch/arm/tools/syscall.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/tools/syscall.tbl b/arch/arm/tools/syscall.tbl
index fbc74b5fa3ed..6a414708ba6c 100644
--- a/arch/arm/tools/syscall.tbl
+++ b/arch/arm/tools/syscall.tbl
@@ -413,3 +413,4 @@
 396	common	pkey_free		sys_pkey_free
 397	common	statx			sys_statx
 398	common	rseq			sys_rseq
+399 common io_pgetevents sys_io_pgetevents
-- 
